storage/innobase/include/sync0policy.h

```diff
@@ -105,7 +105,7 @@ class MutexDebug {
 
 			msg << m_mutex->policy().to_string();
 
-			if (os_thread_pf(m_thread_id) != ULINT_UNDEFINED) {
+			if (os_thread_pf(get_thread_id()) != ULINT_UNDEFINED) {
```

Comment:
This should not be needed. As discussed it is supposed to be protected by ```synch_check_*()``` stuff. Or you disagree?

Reply:
Well, I don't have a data race report on this code. And thus it's ok to do unsynchronized read here.